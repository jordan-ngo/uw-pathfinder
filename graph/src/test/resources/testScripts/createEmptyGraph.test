# Create a graph named empty
CreateGraph empty

# Output all of the nodes in this empty graph
ListNodes empty