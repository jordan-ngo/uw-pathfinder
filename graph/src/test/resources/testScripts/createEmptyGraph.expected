# Create a graph named empty
created graph empty

# Output all of the nodes in this empty graph
empty contains: